apps/prairielearn/src/pages/partials/instructorInfoPanel.ejs

```diff
@@ -43,15 +43,16 @@
   <div class="d-flex flex-wrap">
     <div class="pr-1">QID:</div>
     <div>
+      <% const query = typeof variant !== "undefined" ? `?variant_seed=${variant.variant_seed}` : "" %>
       <% if (locals.course_instance) { %>
-      <a href="<%= plainUrlPrefix %>/course_instance/<%= course_instance.id %>/instructor/question/<%= question.id %>?variant_seed=<%= variant.variant_seed %>"><%= question.qid %></a>
+      <a href="<%= plainUrlPrefix %>/course_instance/<%= course_instance.id %>/instructor/question/<%= question.id %><%= query %>"><%= question.qid %></a>
       <% } else { %>
-      <a href="<%= plainUrlPrefix %>/course/<%= course.id %>/question/<%= question.id %>?variant_seed=<%= variant.variant_seed %>"><%= question.qid %></a>
+      <a href="<%= plainUrlPrefix %>/course/<%= course.id %>/question/<%= question.id %><%= query %>"><%= question.qid %></a>
       <% } %>
     </div>
   </div>
 
-  <% if (question_is_shared && course.sharing_name) { %>
+  <% if (typeof question_is_shared !== "undefined" && question_is_shared && course.sharing_name) { %>
```

Comment:
It's not a dealbreaker, but ideally this should be shown in this case.

Reply:
Sorry, hit enter by mistake. Meant to say: ideally this should be shown, but there is a path to get this info by clicking on the question above it. Since the variant content isn't shown I don't see this being a problem.